executor/android/x86_64_app_policy.h

```diff
@@ -4,121 +4,115 @@
 // File autogenerated by genseccomp.py from Android U - edit at your peril!!
```

Comment:
Nit: I would keep `genseccomp.py` arguments somewhere in the comments (if there's nothing non-public, of course). It would definitely simplify the life of someone who would want to edit this file later.

Reply:
Sounds good! I added a link to the build file with the Android build arguments, in case these change over time.